src/utils/src/arg_parser.rs

```diff
@@ -330,10 +331,10 @@ impl<'a> Arguments<'a> {
         let (args, extra_args) = Arguments::split_args(&args[1..]);
         self.extra_args = extra_args.to_vec();
 
-        // If `--help` is provided as a parameter, we artificially skip the parsing of other
+        // If `--help` or `-h`is provided as a parameter, we artificially skip the parsing of other
         // command line arguments by adding just the help argument to the parsed list and
         // returning.
-        if args.contains(&HELP_ARG.to_string()) {
+        if args.contains(&HELP_ARG.to_string() || &SHORT_HELP_ARG.to_string()) {
```

Comment:
I think what you want to express here is

```suggestion
        if args.contains(&HELP_ARG.to_string()) || args.contains(&SHORT_HELP_ARG.to_string()) {
```

but that wouldn't really work because then even something like `./jailer --hello` would end up printing the help text :/

Reply:
excuse the very dumb mistake 😅

Reply:
@roypat thank you for the PR review!
Based on your comment here I attempted to do some more due dilligence and tried to replicate the logic of the ArgParser to see if this would work as intended.
I added the program I used and its output in the PR description. ptal.

I also added this to the changelog.

Reply:
ah, my bad, I was misreading the code. `args` is a vec of strings, not a string itself. Yes, then this should work! 

Could you add a unit test as well? :o